src/main/java/crud/storage/Storage.java

```diff
@@ -0,0 +1,34 @@
+package crud.storage;
+
+import crud.model.Car;
+import crud.model.Driver;
+import crud.model.Manufacturer;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static Map<Long, Car> carsStorage = new HashMap<>();
+    public static Map<Long, Driver> driversStorage = new HashMap<>();
+    public static Map<Long, Manufacturer> manufacturersStorage = new HashMap<>();
```

Comment:
```suggestion
    public static Map<Long, Manufacturer> manufacturers = new HashMap<>();
```
the same with others

Reply:
https://github.com/Max-Pochepets/jv-create-annotation/pull/2#discussion_r554951749